.buildkite/pipeline.build_core.yml

```diff
@@ -1,10 +1,10 @@
 #ci:group=:book: core tests (civ2)
 
+- label: ":python: small & medium"
```

Comment:
this label here also feels weird, probably needs more rebase/merge audit?

Reply:
fixed now